matrix/common/util.py

```diff
@@ -0,0 +1,42 @@
+import pandas
+import hca
+import zarr
+
+
+def get_client(dss_instance=None):
```

Comment:
What do you think of having this as a @property in `HCAStore`? I think it comes down to, do we know if other parts of matrix service are going to need to access the DSSClient outside of `HCAStore`?